### PR TITLE
New version: TheDocumentFoundation.LibreOffice.LTS version 7.6.4

### LINKED ISSUE #138471: [Update Request]: LibreOffice.LTS 7.6.4
### What type of update are you requesting?

A new version of an existing package

### Current Package Identifier

TheDocumentFoundation.LibreOffice.LTS

### Package Version

7.6.4.1

### Please describe the changes you would like to see

Current LTS version is 7.6.4, installer shows "7.6.4.1". It is probably identical to https://github.com/microsoft/winget-pkgs/tree/master/manifests/t/TheDocumentFoundation/LibreOffice/7.6.4.1 (non-LTS)? I'm honestly a bit confused about the versioning.

Downloads Page: https://www.libreoffice.org/download/download-libreoffice/
Download x86_64: https://download.documentfoundation.org/libreoffice/stable/7.6.4/win/x86_64/LibreOffice_7.6.4_Win_x86-64.msi
Download x86: https://download.documentfoundation.org/libreoffice/stable/7.6.4/win/x86/LibreOffice_7.6.4_Win_x86.msi
Download ARM64: https://download.documentfoundation.org/libreoffice/stable/7.6.4/win/aarch64/LibreOffice_7.6.4_Win_aarch64.msi

### PR DESCRIPTION
**Pull request by CoolPlayLin**
- Resolves #138471
 ###### Microsoft Reviewers: [Open in CodeFlow](https://microsoft.github.io/open-pr/?codeflow=https://github.com/microsoft/winget-pkgs/pull/138502)

### CODE REVIEW

**Comment by Mat2095** [comment]
Hm, could this lead to issues, if two packages (`TheDocumentFoundation.LibreOffice` and `TheDocumentFoundation.LibreOffice.LTS`) both have the same version of the same application, meaning identical installer, product-code and so on? At the moment there is no overlap in versions yet.

**Comment by stephengillie** [comment]
Hi @Trenly and @mdanish-kh,

What are your thoughts on having duplicate installer details in 2 different `PackageIdentifier`s, as described by @Mat2095?

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.installer.yaml [hunk 1]
```diff
@@ -0,0 +1,56 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.installer.1.5.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.6.4
```
Comment (mdanish-kh) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.installer.yaml:5]:
```suggestion
PackageVersion: 7.6.4.1
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.locale.en-US.yaml [hunk 2]
```diff
@@ -0,0 +1,23 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.defaultLocale.1.5.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.6.4
```
Comment (mdanish-kh) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.locale.en-US.yaml:5]:
```suggestion
PackageVersion: 7.6.4.1
```

File: manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.yaml [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Created using wingetcreate 1.5.7.0
+# yaml-language-server: $schema=https://aka.ms/winget-manifest.version.1.5.0.schema.json
+
+PackageIdentifier: TheDocumentFoundation.LibreOffice.LTS
+PackageVersion: 7.6.4
```
Comment (mdanish-kh) [inline manifests/t/TheDocumentFoundation/LibreOffice/LTS/7.6.4/TheDocumentFoundation.LibreOffice.LTS.yaml:5]:
```suggestion
PackageVersion: 7.6.4.1
```

**Review by mdanish-kh — CHANGES_REQUESTED** [review]
After committing the suggested changes, you would also have to rename the version folder to match the new PackageVersion. The folder under `manifests/t/TheDocumentFoundation/LibreOffice/LTS` should be renamed from `7.6.4` -> `7.6.4.1`

A quick and easy way to do this would be here in GitHub Web by pressing the <kbd>.</kbd> (dot) key on your keyboard that'll open up the PR in VS Code Online. Navigate to Explorer menu, find the required folder and make the appropriate rename changes.

After that, stage and commit your changes from Source Control menu on the left. (The changes are automatically pushed)

**Comment by mdanish-kh** [comment]
> Hi @ Trenly and @ mdanish-kh,
> 
> What are your thoughts on having duplicate installer details in 2 different `PackageIdentifier`s, as described by @ Mat2095?

If I'm understanding this correctly - `7.6.4.1`  has recently become an LTS version? If that is the case, we can consider removing it first from `TheDocumentFoundation.LibreOffice` package id, and then merge in this PR.

**Comment by Mat2095** [comment]
> If I'm understanding this correctly - `7.6.4.1` has recently become an LTS version? If that is the case, we can consider removing it first from `TheDocumentFoundation.LibreOffice` package id, and then merge in this PR.

Yes, `7.6.4.1` is the current LTS version (since ~2 weeks).
Could removing this version from `TheDocumentFoundation.LibreOffice` lead to an issue like the following?
- user installed `TheDocumentFoundation.LibreOffice` `7.6.4.1` in the past
- it gets removed and added to `TheDocumentFoundation.LibreOffice.LTS`
- user searches for updates and now receives updates from `TheDocumentFoundation.LibreOffice.LTS` instead of `TheDocumentFoundation.LibreOffice`

**Comment by mdanish-kh** [comment]
> Yes, `7.6.4.1` is the current LTS version (since ~2 weeks). Could removing this version from `TheDocumentFoundation.LibreOffice` lead to an issue like the following?
> 
>     * user installed `TheDocumentFoundation.LibreOffice` `7.6.4.1` in the past
> 
>     * it gets removed and added to `TheDocumentFoundation.LibreOffice.LTS`
> 
>     * user searches for updates and now receives updates from `TheDocumentFoundation.LibreOffice.LTS` instead of `TheDocumentFoundation.LibreOffice`

Ah yes, I believe that is going to be a side-effect of this change. Not sure what's more valuable - keeping existing users to get releases from the same "channel" they installed from, or correctly reflecting the status of each version in the repository. I'm inclining towards the second option, not the most ideal for existing users, but `winget list libreoffice` will help them see that they're now on the LTS version. cc @Trenly thoughts?

I think the actual solution would be implementation of proper channel support in winget:

- https://github.com/microsoft/winget-cli/issues/147

**Comment by Mat2095** [comment]
@mdanish-kh Well, in the meantime 7.6.5 has been released. That version does not exist (and never existed) for the `TheDocumentFoundation.LibreOffice` package, so it does not create any issues. So I will close my issue for 7.6.4 LTS and hopefully I can manage to create a PR myself :)

**Pull request closed without merging**